portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,6 +27,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Lindsey!</h1>");
```

Comment:
Thank you for the detailed pull request description! "Confirm servlet connection" can mean anything, but the continuation really helped identify what I should be looking for as a reviewer.

Reply:
Great! Glad to be starting to get the hang of pull requests :)